tests/unittests/CMakeLists.txt

```diff
@@ -330,7 +325,7 @@ if(GLOW_WITH_OPENCL)
                           ExecutionEngine
                           Graph
                           IR
-                          Optimizer
+                          OpenCLBackend
```

Comment:
this might be it.
As @opti-mix mentioned order of libs matters for linux linker. There might be two entries for oclbackend here, you could check with: ninja -v the exact command.

Reply:
It seems to work on both if I include Backends instead of OpenCLBackend. Does that seem sane?

Reply:
Yeah, that sounds good.